composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": "^5.3.9 || ^7.0",
+        "php": "^5.5 || ^7.0",
```

Comment:
@core23 The lowest PHP version among major distributions is 5.5.9. Travis too uses 5.5.9 as minimum. FOSRestBundle does it as well (https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/composer.json#L24) etc.